integration/spark2/src/main/scala/org/apache/carbondata/store/SparkCarbonStore.scala

```diff
@@ -110,8 +110,16 @@ class SparkCarbonStore extends MetaCachedCarbonStore {
   }
 
   def startSearchMode(): Unit = {
+    LOG.info("Search mode master start service")
     master = new Master(session.sparkContext.getConf)
     master.startService()
+    var count = 0
+    while (!master.isStarted()) {
+      Thread.sleep(10)
+      count = count + 1;
+      LOG.info("Sleeping and waiting search mode master finish start " + count)
```

Comment:
This log should be printed before action (line 118)

Reply:
ok